client/src/components/Settings/SettingsPrivacyView.res

```diff
@@ -7,86 +7,41 @@ module T = SettingsPrivacy
+module C = SettingsViewComponents
 
-let disableOmniOpen = EventListeners.nothingMouseEvent("mousedown")
+let explanation = "To help us understand how people learn Dark, is it okay if we track your session in a replayable format (using Fullstory)."
 
-let radio = (~value: string, ~label: string, ~msg: Settings.msg, ~checked: bool): Html.html<
-  AppTypes.msg,
-> => {
-  let key = "fs-consent-" ++ value
-  Html.div(
-    list{Attrs.class("choice"), disableOmniOpen},
-    list{
-      Html.input(
-        list{
-          Attrs.type'("radio"),
-          Attrs.id(key),
-          Attrs.name("fs-consent"),
-          Attrs.value(value),
-          Attrs.checked(checked),
-          EventListeners.eventNoPropagation(~key, "click", _ => Msg.SettingsMsg(msg)),
-        },
-        list{},
+let viewFSConsent = (checked: bool): Html.html<AppTypes.msg> => {
+  let toggle = {
+    let attr = EventListeners.eventNoPropagation(
+      ~key = `toggle-fs-consent-${string_of_bool(checked)}`,
+      "click",
+      _ => Msg.SettingsMsg(
+        Settings.PrivacyMsg(SettingsPrivacy.SetRecordConsent(!checked))
       ),
-      Html.label(list{Attrs.for'(key)}, list{Html.text(label)}),
-    },
-  )
-}
-
-let consentRow = (recordConsent: option<bool>, ~longLabels: bool): Html.html<AppTypes.msg> => {
-  let (yes, no) = if longLabels {
-    ("Yes, please go ahead", "No, please don't")
-  } else {
-    ("Yes", "No")
+    )
+    C.toggleButton(attr, checked)
   }
 
-  Html.div(
-    list{Attrs.class("setting-row")},
-    list{
-      Html.div(
-        list{Attrs.class("setting-label")},
-        list{
-          Html.div(list{Attrs.class("title")}, list{Html.text("Record me using Dark")}),
-          Html.div(list{Attrs.class("description")}, list{Html.text(explanation)}),
-        },
-      ),
-      Html.div(
-        list{Attrs.class("setting-control")},
-        list{
-          radio(
-            ~value="yes",
-            ~label=yes,
-            ~msg=Settings.PrivacyMsg(SettingsPrivacy.SetRecordConsent(true)),
-            ~checked=recordConsent == Some(true),
-          ),
-          radio(
-            ~value="no",
-            ~label=no,
-            ~msg=Settings.PrivacyMsg(SettingsPrivacy.SetRecordConsent(false)),
-            ~checked=recordConsent == Some(false),
-          ),
-        },
-      ),
-    },
+  Html.div(list{Attrs.class(%twc("mt-10"))}, list{
+    C.settingRow(
+      ~info=None,
+      ~error=None,
+      "",
+      list{
+        Html.div(list{Attrs.class(%twc("flex"))}, list{
+        Html.span(list{}, list{
+          C.sectionHeading("Record me using Dark", None),
+          C.sectionIntroText(list{Html.text(explanation)})
+        }),
+        Html.span(list{Attrs.class(%twc("mt-6"))},list{toggle})
+        })
+      }
+    )}
   )
 }
 
 let view = (state: T.t): list<Html.html<AppTypes.msg>> => {
-  list{consentRow(state.recordConsent, ~longLabels=false)}
+  list{viewFSConsent(state.recordConsent|> Belt.Option.getExn)}
```

Comment:
We shouldn't be using getExn here. I guess we should make recordConsent a bool and default it to `true` - we don't use `None` anymore